### ISSUE TITLE
BankMoney getting wiped

### ISSUE THREAD

**Issue by WinstonSmith-UKGZ** [micovery/ArmA3_Wasteland.Altis#22]
Very impressed, I originally set this up to use mysql which worked out of the box. but I've added an ATM script which saved to the DB but the BankMoney value is not carried over when the scope is read and the .bak is created. I've also tested this using couch and its the same issue. If I go and look in the .bak the expected Bankmoney is written there but in game its 0 and the new scope is 0. By editing line 292 of pFunctions.sqf to:  case "BankMoney": { player setVariable ["bmoney", OR(_value,0), true] };  worked for me.

**Comment by micovery** [comment]
Hey Winston, That's awesome that you are using  it :smile: ! That line you changed, I imagine that the issue was the missing `true` at the end?

When the player's bank money is restored the first time it would probably work, but after a restart, it would be saved as 0 ... because the value is not being broadcast across the network.

I am not sure how the ATM addon works, but I imagine that anytime the player deposits, or withdraws from the bank  it would do a similar  `setVariable` call with the `true` argument.

I'll make this change to the mission. 

Basically, I put this "bmoney" variable in-there after suggestion from another admin who knew about other servers that use the ATM addon.  However since the vanilla mission does not have ATM, I never could fully test it. Do you have your mission o git, or somewhere else I can take look.

**Comment by micovery** [comment]
Fixed by a00851f309744b870eb0628c47c665b129a90e27
